openpower/package/openpower-pnor/Config.in

```diff
@@ -12,6 +12,22 @@ config BR2_PACKAGE_OPENPOWER_PNOR
         help
             Utilites for building a targeting binary image
 
+config BR2_OPENPOWER_SECUREBOOT_KEY_TRANSITION
+        string "Type of Transition HW key hash"
+        help
+            Available options [imprint | production]
```

Comment:
In order to enforce the requirements of 'imprint' or 'production', instead of string, you can use a 'choice' directive, and then the buildroot UI will render things in a nice either-or bullet list.

I'm not entirely convinced that this is the best way to represent how transitions may work... but I don't see that as a blocker for this going in.